apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -1078,7 +1124,8 @@ static void tc_fs_vfs_sendfile(void)
 	TC_ASSERT_GEQ_CLEANUP("open", fd2, 0, close(fd1));
 
 	ret = lseek(fd1, 0, SEEK_END);
-	TC_ASSERT_EQ_CLEANUP("lseek", ret, size, close(fd1); close(fd2));
+	TC_ASSERT_EQ_CLEANUP("lseek", ret, size, close(fd1);
+						 close(fd2));
```

Comment:
Why do you make this change?

Reply:
Fixed. Sorry. This change was done by the formatter tool.